ray_on_golem/server/services/golem/golem.py

```diff
@@ -255,7 +307,7 @@ async def _create_stack(
         return stack
 
     async def _get_proposal_expiration(self, proposal: Proposal) -> timedelta:
-        return await proposal.get_expiration_date() - datetime.now(timezone.utc)
+        return (await proposal.get_expiration_date() - datetime.now(timezone.utc)) * 0.8
```

Comment:
where does the `0.8` come from? wouldn't it make sense to define this magic number as a sensibly named constant so it's obvious what it does?